crispycuts/src/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+// import reportWebVitals from './reportWebVitals';
+
+// ReactDOM.render(
+//   <React.StrictMode>
+//     <App />
+//   </React.StrictMode>,
+//   document.getElementById('root')
+// );
+
+// // If you want to start measuring performance in your app, pass a function
+// // to log results (for example: reportWebVitals(console.log))
+// // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+// reportWebVitals();
+
+const Body= ()=>{
+  return(
+    <div id="body">
+      <Header />
+      <Card 
+        className='section'
```

Comment:
Where's this className defined? Class name is how you use css classes. Like `class` in normal css (they just couldn't use `class`, since `class` is a reserved keyword in javascript.